airflow/configuration.py

```diff
@@ -243,7 +243,11 @@ def _validate_config_dependencies(self):
             # Some of the features in storing rendered fields require sqlite version >= 3.15.0
             min_sqlite_version = '3.15.0'
             if StrictVersion(sqlite3.sqlite_version) < StrictVersion(min_sqlite_version):
-                raise AirflowConfigException(f"error: cannot use sqlite version < {min_sqlite_version}")
+                raise AirflowConfigException(
+                    f"error: cannot use sqlite version < {min_sqlite_version}. "
+                    f"See https://github.com/apache/airflow/blob/master/docs/apache-airflow/"
+                    f"howto/set-up-database.rst#sqlite-database-setup"
```

Comment:
```suggestion
                    f"See {docs_url("howto/set-up-database.rst#sqlite-database-setup")}"
```

(`from airflow.utils.docs import docs_url`)